network/network.go

```diff
@@ -0,0 +1,68 @@
+package network
+
+import (
+	"golang.org/x/net/context"
+
+	"github.com/dedis/cothority/monitor"
+)
+
+// Conn is the basic interface to represent any communication mean
+// between two host.
+type Conn interface {
+	// Send a message through the connection.
+	// obj should be a POINTER to the actual struct to send, or an interface.
+	// It should not be a Golang type.
+	Send(ctx context.Context, obj Body) error
```

Comment:
If we don't want this flexibility: "Context type, which carries deadlines, cancelation signals, and other request-scoped values across API boundaries and between processes." 
(which we currently don't seem to use at all), I'm OK with removing it.